### DISCUSSION TITLE
[Request] 增加使用同步功能的文档

### DISCUSSION THREAD

**Discussion by Vateer** [lobehub/lobehub#2276] [General Question] [answered]
### 🥰 需求描述

我翻看了一下issues，发现关于WebRTC Sync的bug报告存在，并且我自己在使用中也确实无法同步。
我的使用方法是两台设备填入相同的Sync Channel Name和Sync Channel Password，同时在Device Information中填入不同的名字，但遗憾无法同步。
假如项目确实支持同步功能，恳请项目组增加对应的使用文档。

### 🧐 解决方案

恳请项目组增加对应的使用文档

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment, marked as answer]
目前看下来存在的问题主要出现在几个方面：

- 由于网络不支持 WebRTC 协议（防火墙屏蔽等），导致无法同步
- iPhone 设备似乎经常无法同步
- 由于缺少中心化的节点，导致同步信息可能会存在互相覆盖的问题

所以目前看下来之前做的 WebRTC 的方案不能作为稳定的数据同步方案，后续应该也就只会处于「实验性」阶段，所以也不准备补充相关文档了。如果有数据同步需求，请关注 https://github.com/lobehub/lobe-chat/discussions/1851 进展

**Reply by ous50** [reply]
我在本地网络的两台电脑之间都没法用 WebRTC 同步23333

**Reply by arvinxx** [reply]
@ous50 不能同步大概率是网络问题
